### PR TITLE
Part of #17670 for file custom-spending-cap-tooltip.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a Part of #17670 Replace Typography with Text component.

In this PR I have worked on the file
- **ui\components\app\custom-spending-cap\custom-spending-cap-tooltip.js**

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-  TypographyVariant,
+  TextVariant,
 } from '../../../helpers/constants/design-system';
-import { Icon, IconName, IconSize } from '../../component-library';
+import {
+  Icon,
+  ICON_NAMES,
+  ICON_SIZES,
+} from '../../component-library/icon/deprecated';
+import { Text } from '../../component-library';
```
Comment (georgewrmarshall) [inline ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js:15]: We are currently in the process of migrating to TypeScript we want to use the enums over the constants

```suggestion
import {
  Icon,
  IconName,
  IconSize,
  Text
} from '../../component-library';
```

Comment (georgewrmarshall) [reply ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js:15]: This may have some formatting issues that need linting

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, just need to fix the imports but otherwise looks good!

**Comment by PrgrmrHarshShukla** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by pedronfigueiredo — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**